src/commonMain/kotlin/app/cash/turbine/channel.kt

```diff
@@ -64,16 +77,51 @@ public fun <T> ReceiveChannel<T>.expectNoEvents() {
  *
  * This function will always return a terminal event on a closed [ReceiveChannel].
  */
-public suspend fun <T> ReceiveChannel<T>.awaitEvent(): Event<T> =
-  try {
-    Event.Item(receive())
+@OptIn(ExperimentalCoroutinesApi::class)
```

Comment:
What is this for?

Reply:
To silence a warning on `coroutineContext[TestCoroutineScheduler]`. Unsure why it then complains that the `@OptIn` is unnecessary.